ci-operator/config/openshift/ci-tools/openshift-ci-tools-master__arm64.yaml

```diff
@@ -0,0 +1,558 @@
+base_images:
+  os:
+    name: centos
+    namespace: ocp
+    tag: stream8
+binary_build_commands: make production-install
+build_root:
+  from_repository: true
+  use_build_cache: true
```

Comment:
Removing this could fix the `pj-rehearse` build cache distribution error. Using the build cache is not necessary and the build cache image will not work on ARM anyway.

Reply:
thanks, trying that now